jme3-lwjgl3/build.gradle

```diff
@@ -0,0 +1,12 @@
+if (!hasProperty('mainClass')) {
+    ext.mainClass = ''
+}
+
+dependencies {
+    compile project(':jme3-core')
+    compile project(':jme3-desktop')
+    compile 'org.lwjgl:lwjgl:3.0.0b-SNAPSHOT'
```

Comment:
Basically the 3.0.0b has some api changes in it to do with creating the GL capabilities and since this is more of an experiment module for now I thought having the 3.0.0b snapshot would be OK in order to get those api changes in place.